### PR TITLE
6305 National Party Accounts Types Filters Labels

### LINKED ISSUE #6305: Update the wording on the national party account data table filter for transaction type
RELATED TICKET: https://github.com/fecgov/fec-cms/issues/6301

### What we're after:
Organize the transaction type filter options on the National Party Account data tables to be more user friendly. Currently, the filters list transaction types that we used to populate the data in the table, those transaction type descriptions aren't consistently worded which makes the list difficult to read.

Instead, format the filter menu to appear like the Receipt Type menu on the[ receipts table](https://www.fec.gov/data/receipts/?data_type=processed&two_year_transaction_period=2024&min_date=01%2F01%2F2023&max_date=12%2F31%2F2024), making use of the headings within the menu: 

image

For the National Party Account menu the headings would be the account names, something like this:

> More
> **Headquarters account**
>    &nbsp;&nbsp;&nbsp;Individual, partnership, limited liability company
>    &nbsp;&nbsp;&nbsp;Earmarked receipt
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from individual 
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from registered filer
>    &nbsp;&nbsp;&nbsp;Transfer
>    &nbsp;&nbsp;&nbsp;Registered filer
>    &nbsp;&nbsp;&nbsp;Native American tribe
> 
> **Recount account**
>    &nbsp;&nbsp;&nbsp;Individual, partnership, limited liability company
>    &nbsp;&nbsp;&nbsp;Earmarked receipt
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from individual 
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from registered filer
>    &nbsp;&nbsp;&nbsp;Transfer
>    &nbsp;&nbsp;&nbsp;Registered filer
>    &nbsp;&nbsp;&nbsp;Native American tribe
>
> **Convention account**
>    &nbsp;&nbsp;&nbsp;Individual, partnership, limited liability company
>    &nbsp;&nbsp;&nbsp;Earmarked receipt
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from individual 
>    &nbsp;&nbsp;&nbsp;Joint fundraising receipt from registered filer
>    &nbsp;&nbsp;&nbsp;Transfer
>    &nbsp;&nbsp;&nbsp;Registered filer
>    &nbsp;&nbsp;&nbsp;Native American tribe

### Related issues

- #6179
- #6180

### PR DESCRIPTION
**Pull request by rfultz**
## Summary

- Resolves #6305 

Updating the labels for the type filters for the National Party Accounts datatables

### Required reviewers

- UX?
- 1 Front-end
- SME

## Impacted areas of the application

The national party accounts datatables' [receipts | disbursements] type filters

## Screenshots
Disbursements
image

Receipts
image

## Related PRs

None

## How to test

- pull the branch
- `./manage.py runserver`
- check [national party accounts disbursements](http://127.0.0.1:8000/data/national-party-account-disbursements/)' disbursement type filter (left column, Disbursement details)
- check [national party accounts receipts](http://127.0.0.1:8000/data/national-party-account-receipts/)' receipt type filter (left column, Receipt details)

### CODE REVIEW

File: fec/data/templates/macros/filters/dropdown-json.jinja [hunk 1]
```diff
@@ -59,7 +59,7 @@ data-validate="false" data-modifies-filter="{{ name }}" data-required-default="{
 <div class="filter js-filter js-filter-control"  data-filter="select" data-name="{{ name }}" data-validate="false" data-modifies-filter="{{ name }}">
   <label for="{{ name }}" class="label">{{ label }}</label>
   <label class="label--help u-margin--top">Limit 1 {{ label | lower }}</label>
-  <select id={{ name }} name="{{ name }}" data-filter-change="true" data-removeable-filter="true">
+  <select id={{ name }} name="{{ name }}" data-filter-change="true" data-removeable-filter="true" class="dropdown">
```
Comment (johnnyporkchops) [inline fec/data/templates/macros/filters/dropdown-json.jinja:62]: It look like this is a mistake that got carried over every time we repurposed the original dropdown macro, shouldn't it be `select id="{{ name }}"` , with quotes? Every macro in this file has that.

**Review by johnnyporkchops — CHANGES_REQUESTED** [review]

File: fec/data/templates/macros/filters/dropdown-json.jinja (hunk 1 above)
Comment (rfultz) [reply fec/data/templates/macros/filters/dropdown-json.jinja]: Oh, nice catch! I've fixed a few of these in the latest push

**Review by johnnyporkchops — APPROVED** [review]
Looks good, thanks for making that one change!

**Review by bmathesonFEC — APPROVED** [review]
Just huddled with Jonella. Looks great, huge improvement on the codes/code wording. Thank you!

**Comment by JonellaCulmer** [comment]
@rfultz My only concern is the low contrast between the header labeling and background.

**Review by JonellaCulmer — APPROVED** [review]
Approving under the assumption that the contrast issue will be resolved shortly.

**Pull request merged**